fuzzers/inprocess/libfuzzer_libpng_accounting/src/lib.rs

```diff
@@ -21,246 +18,197 @@ use libafl::{
     state::{HasCorpus, StdState},
     Error, HasMetadata,
 };
 use libafl_bolts::{
-    core_affinity::Cores,
     rands::StdRand,
-    shmem::{ShMemProvider, StdShMemProvider},
     tuples::{tuple_list, Merge},
     AsSlice,
 };
-use libafl_targets::{
-    libfuzzer_initialize, libfuzzer_test_one_input, ACCOUNTING_MEMOP_MAP, EDGES_MAP,
-    MAX_EDGES_FOUND,
-};
+use libafl_targets::{libfuzzer_initialize, libfuzzer_test_one_input, EDGES_MAP, MAX_EDGES_FOUND};
 use mimalloc::MiMalloc;
 
 #[global_allocator]
 static GLOBAL: MiMalloc = MiMalloc;
 
-/// Parse a millis string to a [`Duration`]. Used for arg parsing.
-fn timeout_from_millis_str(time: &str) -> Result<Duration, Error> {
-    Ok(Duration::from_millis(time.parse()?))
-}
-
-/// The commandline args this fuzzer accepts
-#[derive(Debug, Parser)]
-#[command(
-    name = "libfuzzer_libpng_launcher",
-    about = "A libfuzzer-like fuzzer for libpng with llmp-multithreading support and a launcher",
-    author = "Andrea Fioraldi <andreafioraldi@gmail.com>, Dominik Maier <domenukk@gmail.com>"
-)]
-struct Opt {
-    #[arg(
-    short,
-    long,
-    value_parser = Cores::from_cmdline,
-    help = "Spawn a client in each of the provided cores. Broker runs in the 0th core. 'all' to select all available cores. 'none' to run a client without binding to any core. eg: '1,2-4,6' selects the cores 1,2,3,4,6.",
-    name = "CORES"
-    )]
-    cores: Cores,
-
-    #[arg(
-        short = 'p',
-        long,
-        help = "Choose the broker TCP port, default is 1337",
-        name = "PORT",
-        default_value = "1337"
-    )]
-    broker_port: u16,
-
-    #[arg(short = 'a', long, help = "Specify a remote broker", name = "REMOTE")]
-    remote_broker_addr: Option<SocketAddr>,
-
-    #[arg(
-        short,
-        long,
-        help = "Set an initial corpus directory",
-        name = "INPUT",
-        required = true
-    )]
-    input: Vec<PathBuf>,
-
-    #[arg(
-        short,
-        long,
-        help = "Set the output directory, default is ./out",
-        name = "OUTPUT",
-        default_value = "./out"
-    )]
-    output: PathBuf,
-
-    #[arg(
-    value_parser = timeout_from_millis_str,
-    short,
-    long,
-    help = "Set the execution timeout in milliseconds, default is 10000",
-    name = "TIMEOUT",
-    default_value = "10000"
-    )]
-    timeout: Duration,
-    /*
-    /// This fuzzer has hard-coded tokens
-    #[arg(
-
-        short = "x",
-        long,
-        help = "Feed the fuzzer with an user-specified list of tokens (often called \"dictionary\"",
-        name = "TOKENS",
-        multiple = true
-    )]
-    tokens: Vec<PathBuf>,
-    */
-}
-
-/// The main fn, `no_mangle` as it is a C symbol
+/// The main fn, `no_mangle` as it is a C main
 #[no_mangle]
 pub extern "C" fn libafl_main() {
     // Registry the metadata types used in this fuzzer
     // Needed only on no_std
     // unsafe { RegistryBuilder::register::<Tokens>(); }
-    let opt = Opt::parse();
```

Comment:
Why did you remove opt parsing?

Reply:
Oh, …, ehm, …, that shouldn't have been removed.

Not sure what happened there. I guess you shouldn't program late at night.